features/suzie_views_a_petition.feature

```diff
@@ -94,3 +94,15 @@ Feature: Suzie views a petition
     When I view the petition
     Then I should see "Parliament debated this petition on 26 October 2015"
     And I should see "Waiting for 1 day for Parliament to publish the debate outcome"
+
+  Scenario: Suzie does not see information about future signature targets when viewing a closed petition
+    Given a petition "Spend more money on Defence" has been closed
+    When I view the petition
+    Then I should not see "At 10,000 signatures..."
+    Then I should not see "At 100,000 signatures..."
+
+  Scenario: Suzie sees information about future signature targets when viewing an open petition
+    Given an open petition "Spend more money on Defence"
+    When I view the petition
+    Then I should see "At 10,000 signatures..."
+    Then I should see "At 100,000 signatures..."
```

Comment:
Again you probably want to add some scenarios for open petitions over 10,000 hiding the first block and then over 100,000 hiding the second block and showing the response and debate information.